File: Q7/Q7.js
const rows = parseInt(prompt("Enter the number of rows: "));
for (let i = 1; i <= rows; i++) 
{
    let row = " ".repeat(rows - i);
    for (let k = 1; k <= i; k++) 
    {
        row += k + " ";
    }
    console.log(row.trimEnd());
}

File: Q7/Q7.c
#include <stdio.h>

int main()
{
    int rows, i, j, k;
    printf("Enter the number of rows: ");
    scanf("%d", &rows);
    for (i = 1; i <= rows; i++) 
    {
        for (j = i; j < rows; j++) 
        {
            printf(" ");
        }
        for (k = 1; k <= i; k++)
          {
            printf("%d ", k);
        }
        printf("\n");
    }
    return 0;
}
